### DISCUSSION TITLE
What will happen to 'dead' PRs?

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#139] [Q&A] [answered]
I was wondering what will happen in situations like #71 where the author seems to have given up or not come back to resolve items & comments?

Are we able to take it forward for them, or do we wait a certain amount of days to expire? I would love to see that feature merged but I'm wondering what we do now?

**Comment by Arkensor** [comment, marked as answer]
That PR is stale until the persistence system is in place so it can be completed and merged. But in general, if the person responsible seems to lack time to continue development and someone else wants to pick it up they could do so. Getting the changes from the pr into your own repo before a merge to main might be a bit tricky.

**Reply by TheOriginalCrazyDuck** [reply]
Less so than you'd think, IIRC you go to the repo of the PR and open a pull request but change the base branch to your own and that should allow you to merge it i to your own repo, there's more verbose info about it on StackOverflow though.
